kramer.py

```diff
@@ -169,7 +169,7 @@ def __init__(self:object,_n1_:{choice(_types_)}=False,_n2_:{choice(_types_)}=0,*
 

```

Comment:
Can we optionally comment this out (I assume this is for fuzzing)?

Reply:
Done!